products.csv

```diff
@@ -0,0 +1,21 @@
+versionId,name,id,version
```

Comment:
Is this a generated file too? Or is it used somewhere? If it's generated, it shouldn't be checked in, and should be ignored, similar to the other build objects